components/ViewHabits.tsx

```diff
@@ -0,0 +1,22 @@
+import { css } from '@emotion/react';
+import Image from 'next/image';
+
+type Props = {};
+
+const ViewHabits = (props: Props) => {
+  return (
+    <div css={containerStyle}>
+      <Image src='/logo.svg' alt='.SHIFT logo' fill />
+    </div>
+  );
+};
```

Comment:
[non-blocking] since this is just a function that returns a value, you might find it slightly more terse to go with 
```suggestion
const ViewHabits = (props: Props) => (
  <div css={containerStyle}>
    <Image src='/logo.svg' alt='.SHIFT logo' fill />
  </div>
);
```

Reply:
Done